app/jobs/bulkrax/create_relationships_job.rb

```diff
@@ -97,19 +97,23 @@ def perform(parent_identifier:, importer_run_id:) # rubocop:disable Metrics/AbcS
       end
 
       if errors.present?
-        # rubocop:disable Rails/SkipsModelValidations
-        importer_run.increment!(:failed_relationships, number_of_failures)
-        # rubocop:enable Rails/SkipsModelValidations
+        ImporterRun.connection.execute(<<-SQL)
+          UPDATE bulkrax_importer_runs
+          SET failed_relationships = COALESCE(failed_relationships, 0) + #{number_of_failures}
+          WHERE id = #{importer_run_id}
+        SQL
 
         parent_entry&.set_status_info(errors.last, importer_run)
 
         # TODO: This can create an infinite job cycle, consider a time to live tracker.
         reschedule({ parent_identifier: parent_identifier, importer_run_id: importer_run_id })
         return false # stop current job from continuing to run after rescheduling
       else
-        # rubocop:disable Rails/SkipsModelValidations
-        Bulkrax::ImporterRun.find(importer_run_id).increment!(:processed_relationships, number_of_successes)
-        # rubocop:enable Rails/SkipsModelValidations
+        ImporterRun.connection.execute(<<-SQL)
```

Comment:
Should this call `ImporterRun.increment_counter(:processed_relationships, importer_run_id, amount: number_of_successes)` ?

I don't like the raw SQL so far from the model.

Reply:
Oops…the increment_counter is a Rails method.  Hmm.

Reply:
<img width="499" alt="Screenshot 2023-10-31 at 12 42 42 PM" src="https://github.com/samvera/bulkrax/assets/2130/5f2b63e8-e1f2-44d2-aa4a-6e9922b251d5">

Wondering about using the `update_counters` method: https://api.rubyonrails.org/classes/ActiveRecord/CounterCache/ClassMethods.html#method-i-update_counters

Reply:
`#update_counters` seems good to me. Same SQL, just gives you control over the amount, which is nice. 

I'm curious, why do you not like raw SQL?

Reply:
does this raw sql work in mysql? all versions of postgres? msql? oracle db? sqlite? because the old version does and that's the reason we avoid raw sql whenever we can.

Reply:
Replaced the raw SQL with `#update_counters` and CI is still green so I think we're good!